examples/interrupt-events.rs

```diff
@@ -0,0 +1,93 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::time::Instant;
+
+use clap::{App, Arg, ArgMatches};
+use colored::*;
+use env_logger;
+
+use microvmi::api::*;
+
+fn parse_args() -> ArgMatches<'static> {
+    App::new(file!())
+        .version("0.1")
+        .about("Watches interrupt VMI events")
+        .arg(Arg::with_name("vm_name").index(1).required(true))
+        .get_matches()
+}
+
+fn get_interrupts(drv: &mut Box<dyn Introspectable>, enabled: bool) {
```

Comment:
nitpick: "get" in the function name usually implies that your function yields a return value. You don't return anything here so this function name is misleading.